plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,2305 @@
+      mac_address:
+        description: |
+          MAC Address field used to identify the device. If MAC address is known,
+          it must be provided. This field cannot be modified.
+          To identify the specific access point, any one of the (mac_address, hostname,
+          management_ip_address) parameters is required.
+          Example: mac_address: "90:e9:5e:03:f3:40"
+        type: str
+        required: True
+      rf_profile:
+        description: |
+          Radio Frequency (RF) profile of the Access Point (e.g., 'HIGH').
+        type: str
+        required: False
+        example: "HIGH"
+      site:
+        description: |
+          Current site details where the Access Point is located.
+        type: object
+        suboptions:
+          floor:
+            description: |
+              Floor details of the current site.
+            type: object
+            required: False
+            suboptions:
+              name:
+                description: |
+                  Name of the floor (e.g., 'FLOOR1').
+                type: str
+                required: False
+              parent_name:
+                description: |
+                  Parent name of the floor in the site hierarchy (e.g., 'Global/USA/New York/BLDNYC').
+                type: str
+                required: False
+      ap_name:
+        description: |
+          Current AP name that needs to be changed along with the new AP name.
+          Example: ap_name: "Test2", ap_name_new: "NFW-AP1-9130AXE"
+        type: str
+        required: False
+      admin_status:
+        description: |
+          Status of the AP configuration. Accepts "Enabled" or "Disabled".
+          Example: admin_status: "Enabled"
+        type: str
+        required: False
+      led_status:
+        description: |
+          State of the AP's LED. Accepts "Enabled" or "Disabled".
+          Example: led_status: "Enabled"
+        type: str
+        required: False
+      led_brightness_level:
+        description: |
+          Brightness level of the AP's LED. Accepts values from 1 to 8.
+          Example: led_brightness_level: 3
+        type: int
+        required: False
+      ap_mode:
+        description: |
+          Mode of operation for the Access Point (AP). Possible values include "local/flexconnect" or "monitor" or "sniffer" or Bridge/Flex+Bridge.
+          - If Ap mode is local/flex, then only radio role assignment can be changed.
+          Example: ap_mode: "local"
+        type: str
+        required: False
+      location:
+        description: |
+          Location name of the AP. Provide this data if a change is required.
+          Example: location: "Bangalore"
+        type: str
+        required: False
+      failover_priority:
+        description: |
+          Priority order for failover in AP configuration. Accepts "Low" or "Medium" or "High" or "Critical".
+        type: str
+        required: False
+      clean_air_si_2.4ghz/5ghz/6ghz:
```

Comment:
can you please split this? If we wanted to add more notes or any other, then it is difficult to do..
```
     clean_air_si_2.4ghz:
        description: |
          Clean Air Spectrum Intelligence (SI) feature status for the 2.4GHz band. Indicates whether Clean Air Spectrum Intelligence is enabled or disabled.
          Example: clean_air_si_2.4ghz: "Enabled"
        type: str
        required: False
      clean_air_si_5ghz:
        description: |
          Clean Air Spectrum Intelligence (SI) feature status for the 5GHz band. Indicates whether Clean Air Spectrum Intelligence is enabled or disabled.
          Example: clean_air_si_5ghz: "Enabled"
        type: str
        required: False
      clean_air_si_6ghz:
        description: |
          Clean Air Spectrum Intelligence (SI) feature status for the 6GHz band. Indicates whether Clean Air Spectrum Intelligence is enabled or disabled.
          Example: clean_air_si_6ghz: "Enabled"
        type: str
        required: False
```

OR 

Make sub options..

```
clean_air_si:
  description: |
    Clean Air Spectrum Intelligence (SI) feature status for different frequency bands.
    Indicates whether Clean Air Spectrum Intelligence is enabled or disabled for each band.
  type: dict
  required: False
  suboptions:
    2.4ghz:
      description: |
        Clean Air Spectrum Intelligence (SI) feature status for the 2.4GHz band.
        Example: 2_4ghz: "Enabled"
      type: str
      required: False
    5ghz:
      description: |
        Clean Air Spectrum Intelligence (SI) feature status for the 5GHz band.
        Example: 5ghz: "Enabled"
      type: str
      required: False
    6ghz:
      description: |
        Clean Air Spectrum Intelligence (SI) feature status for the 6GHz band.
        Example: 6ghz: "Enabled"
      type: str
      required: False

```

Reply:
Updated first one